src/current/v23.2/configure-logs.md

```diff
@@ -483,6 +483,31 @@ sinks:
       channels: {INFO: DEV, ERROR: OPS}
 ~~~
 
+### Set timezone
```

Comment:
Just to clarify, will the datetime/timezone format-options for the json format be handled in a separate PR?

Reply:
Yes, the datetime/timezone format-options for the json format will be handled in a separate PR.

Reply:
Ack, thanks